app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: :post_counter
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] [REQUIRED] Great job so far @gaks1 🥇, however, you seem to have skipped a part of the project requirement that states that the Post model should have a method that updates the posts counter for a user.
### Hint:
```ruby
def update_post_counter
  code here ...
end
```

- [ ] [OPTIONAL] I would like to also suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```ruby
after_save :update_post_counter

  private

  def update_post_counter
   Code here ...
  end
```